presto-main/src/main/java/com/facebook/presto/sql/analyzer/Analysis.java

```diff
@@ -103,6 +105,17 @@ public void setQuery(Query query)
         this.query = query;
     }
 
+    public Optional<Explain> getExplainAnalyze()
+    {
+        return explainAnalyze;
+    }
+
+    public void setExplainAnalyze(Explain explainAnalyze)
```

Comment:
Insert doesn't store the Insert statement. It's just a holder class that contains columns + table information (it's an inner class of Analysis)